prime-dotnet-webapi/Startup.cs

```diff
@@ -56,12 +56,15 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IBusinessEventService, BusinessEventService>();
             services.AddScoped<ISubmissionService, SubmissionService>();
             services.AddScoped<IRazorConverterService, RazorConverterService>();
+            services.AddScoped<ISiteService, SiteService>();
+            services.AddScoped<IPartyService, PartyService>();
 
             services.AddControllers()
                 .AddNewtonsoftJson(options =>
                 {
                     options.SerializerSettings.Converters.Add(new StringEnumConverter(new KebabCaseNamingStrategy(), false));
                     options.SerializerSettings.Converters.Add(new EmptyStringToNullJsonConverter());
+                    options.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore;

```

Comment:
What problem were you seeing that made this necessary?

Reply:
It was because of the old json import so things weren't json ignoring properly. removed now.